resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -241,6 +244,56 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class Scrollbar {
+
+    [DscProperty(Key)] [Bool] $Show = $false
+
+
+

```

Comment:
nit: extra lines.

Reply:
This suggestion contradicts a [previous](https://github.com/microsoft/winget-dsc/pull/49#discussion_r1723879473) suggestion.